ext/standaloneusers/CRM/Standaloneusers/Page/Login.php

```diff
@@ -31,6 +31,9 @@ public function run() {
     // inside the main box.
     $this->assign('statusMessages', CRM_Core_Smarty::singleton()->fetch("CRM/common/status.tpl"));
```

Comment:
I think these lines are now redundant. Could you remove @jaapjansma

Reply:
Also the {$statusMessages} in `Login.tpl`

Reply:
Yes will do that.